src/core/symbology-ng/qgsfillsymbollayer.cpp

```diff
@@ -2093,12 +2095,6 @@ void QgsSVGFillSymbolLayer::toSld( QDomDocument &doc, QDomElement &element, cons
     symbolizerElem.appendChild( doc.createComment( QStringLiteral( "SVG from data not implemented yet" ) ) );
   }
 
-  if ( mSvgOutlineColor.isValid() || mSvgOutlineWidth >= 0 )
```

Comment:
This was removed as the svg outline is not the same thing as the polygon fill outline